README.md

```diff
@@ -1 +1,152 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 portfolio-setup-and-mobile-first <a name="about-project"></a>
+
+>  This is a Capstone project for module one  is developed with microverse requirnment.
+
+**portfolio-setup-and-mobile-first** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The technology uesd in this project is
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://mishdm.github.io/Capston-Project-Module/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+> Still working on its features
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Anyone who wants to work on this project just simple clone to your local machine. this project has been acessed online just for learning propose.
+
+To get a local copy up and running, follow these steps.
+Go to git https://github.com/MishDM/My-Portfolio  and clone My-portfolio repository to your local machine and run the index.html
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Misikir Delelew Mulate**
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Mohammad Mohsen Haidari**
+
+- GitHub: [@github/MishMD](https://github.com/MishMD)
+- Twitter: [@misikirdelelw](https://twitter.com/misikir.delelew)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/misikir-delelew-114a5946)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> features that will be added to the project.
+
+- [ ] **Hero section**
+- [ ] **Navbar**
+- [ ] **featured-in-section**
+- [ ] **Live-demos of projects**
+- [ ] **about-myself**
+- [ ] **contact-form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project... Leave a star on its repository and make me happy 😊
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- > Give credit to everyone who inspired your codebase. -->
+
```

Comment:
- [ ] Your  "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). please update that